khal/khalendar/event.py

```diff
@@ -343,6 +351,27 @@ def organizer(self) -> str:
         else:
             return email
 
+    @property
+    def busystatus(self) -> str:
+        if 'X-MICROSOFT-CDO-BUSYSTATUS' not in self._vevents[self.ref]:
+            return 'BUSY'
```

Comment:
If this property is not set, I don't think we should fall back to `BUSY`, we should return `None` instead. Especially since this property is non-standard.